docs/INSTALLATION.md

```diff
@@ -1,10 +1,10 @@
 ## Dart
 * [Install Dart](https://www.dartlang.org/install)
 * Add the Dart SDK to your path.
-* From a new command window, type `dart --version` to verify that Dart is now available.
+* From a new command window or terminal, type `dart --version` to verify that Dart is now available.
```

Comment:
If they install dart by following steps described at `dart-lang.org/install` there shouldn't be any problem. But it's still possible that someone forgot to update there path variable or perhaps messed up. We should have steps to add dart to path variable.

Reply:
Having steps to add Dart to the Path will need to tackle the Mac, Linux, and Windows versions.

I think mac will be the tricky part since it could have Dart via a tar.gz that was extracted or via HomeBrew (or other auto installation method)